toolchain/driver/testdata/error_codegen_objcode_output_file_with_triple.carbon

```diff
@@ -0,0 +1,8 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
```

Comment:
Let's rename this test to `error_codegen_objcode_no_output_file.carbon` (the triple isn't relevant here, and I think it's useful for the test name to say "no output file" not just "output file").